starter-code/css/main.css

```diff
@@ -6,3 +6,158 @@ Green: #00B172
 White: #FFF
 
 */
+html{
+    font-family: 'Montserrat', sans-serif;
+}
+
+body{
+    margin: auto;
+    padding: 0;
+}
+
+header{
+    display: flex;
+    justify-content: space-between;
+    background-color:white;
+}
+
+.logo {
+    width: auto;
+    height: 50px;
+    margin: 20px;
+}
+
+nav {
+    display: flex;
+    justify-content: space-around;
+    margin: 20px;
+}
+
+ul {
+    display: flex;
+    justify-content: space-between;
+}
+
+a{
+    text-decoration: none;
+    color: #1A1A1A;
+    margin: 0 10px;
+    font-size: 16px;
+    font-weight: 300;
+}
+
+li{
+    display: flex;
+    justify-content: space-around;
+    list-style: none;
+}
+
+#articles{
+    display: flex;
+    justify-content: space-around;
+}
+
+#articles article{
+    text-align: center;
+    margin: 20px 0;
+    width: 20%;
+    height: auto;
+}
+
+#articles p{
+    font-size: 18px;
+    font-weight: 100;
+}
+
+#articles img{
+    width: 50%;
+    height: auto;
+}
+
+#spotlight{
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    padding: 50px;
+    margin: auto;
+    background-color: #00B172;
+    background-image: url("../images/spotify-icon-white.png");
+    background-repeat: no-repeat;
+    background-position: 50% 50%;
+    background-size: 10%;
+    margin: 25px;
+    color: white;
+}
+
+#landing{
+    color: white;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    background-image: url("../images/landing.jpg");
+    background-position: 50% 50%;
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 600px;
```

Comment:
good use of the backgound properties!
To see more of the image, you can increase the height to 1000px for example